package.json

```diff
@@ -95,7 +95,7 @@
     "plotly.js-dist-min": "^1.53.0",
     "reflect-metadata": "^0.1.8",
     "rxjs": "5.4.0",
-    "sanitize-html": "^2.3.2",
+    "sanitize-html": "1.19.1",
```

Comment:
This got bumped due to a security issue so it'd be preferred not to revert this, why is this necessary?

Reply:
The new module doesn't contain a UMD bundle and isn't compatible with webmode.

Reply:
That's unfortunate.

@chlafreniere VS Code seems to use [insane](https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/common/insane/insane.js) for their HTML sanitizing so something we should consider if there's a risk here using the older package.